CHANGELOG.rst

```diff
@@ -1,3 +1,33 @@
+1.1.0 (09-05-2025)
+------------------
+
+**New features**
+
+* [#104] Added filters
+
+    * ProductType themas__naam__in
+    * ProductType themas__uuid__in
+    * Product productype__naam__in
+    * Product eigenaren__bsn
+    * Product eigenaren__kvk_nummer
+    * Product eigenaren__vestigingsnummer
+    * Product eigenaren__klantnummer
+
+**Project maintenance**
+
+* [#20] Added support for django-setup-configuration. see `documentation <https://open-product.readthedocs.io/en/latest/installation/config/openproduct_config_cli.html>`_
```

Comment:
You can make this into a reference, that way it links to the current version of the docs you're reading
```suggestion
* [#20] Added support for django-setup-configuration. see :ref:`documentation<installation_configuration_cli>`
```